dev/Interop/CameraCaptureUI/CameraCaptureUI/CameraCaptureUIPhotoCaptureSettings.cpp

```diff
@@ -43,6 +45,7 @@ namespace winrt::Microsoft::Windows::Media::Capture::implementation
 
     void CameraCaptureUIPhotoCaptureSettings::Serialize(winrt::Windows::Foundation::Collections::ValueSet const& props)
     {
+        auto lock = m_lock.lock_shared();
         props.Insert(L"AllowCropping", box_value(AllowCropping()));
```

Comment:
Use private members like m_allowCropping, m_format etc. directly here as done in other places

Reply:
Done!